.goreleaser.docker.yaml

```diff
@@ -42,25 +42,6 @@ builds:
     goarch:
```

Comment:
Should we switch to what we're doing in https://github.com/abcxyz/jvs instead of using goreleaser? I know this is large already, so feel free to defer or push back :)

Reply:
I'll take this in a follow up.